cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py

```diff
@@ -0,0 +1,2350 @@
+        initialEstimate: 1/2h
+        startsin: 5.6
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_users_rhos7_ga_fedora_22_ext4():
+    """
+    test_ssa_users[rhos7-ga-fedora-22-ext4]
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        caseimportance: medium
+        initialEstimate: 1/2h
+        startsin: 5.3
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_users_scvmm2k16_windows2012r2_ntfs():
+    """
+    Add SCVMM-2016 provider.
+    Perform SSA on Windows 2012 server R2 VM having NTFS filesystem.
+    Check whether it retrieves Users.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+        startsin: 5.6
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_users_scvmm2k12_windows2012r2_ntfs():
+    """
+    Add SCVMM-2012 provider.
+    Perform SSA on Windows 2012 server R2 VM having NTFS filesystem.
+    Check whether it retrieves Users.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+        startsin: 5.6
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_users_ec2_rhel():
+    """
+    Add EC-2 provider.
+    Perform SSA on RHEL instance.
+    Check whether it retrieves Users.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+        startsin: 5.9
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_users_azure_windows2012r2_ntfs():
+    """
+    1. Add Azure provider
+    2. Perform SSA on Windows server 2012 R2.
+    3. Check Users are retrieved.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/3h
+        startsin: 5.6
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_users_ec2_windows2012r2_ntfs():
+    """
+    Add EC-2 provider.
+    Perform SSA on Windows 2012 server R2 VM having NTFS filesystem.
+    Check whether it retrieves Users.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+        startsin: 5.9
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_packages_ec2_ubuntu():
+    """
+    Add EC-2 provider.
+    Perform SSA on Ubuntu instance.
+    Check whether it retrieves Packages.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+        startsin: 5.9
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_packages_scvmm2k12_rhel74():
+    """
+    Add SCVMM-2012 provider.
+    Perform SSA on RHEL 7.4 VM.
+    Check whether Packages retrieved.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+        startsin: 5.6
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_packages_rhos7_ga_fedora_22_ext4():
+    """
+    test_ssa_packages[rhos7-ga-fedora-22-ext4]
```

Comment:
Was this suppose to be a description here?

Reply:
Removed tests already automated.